README.md

```diff
@@ -59,16 +59,17 @@ The crate provides the following types:
 ### Basic Usage
 
 ```rust
-use opentelemetry::sdk::trace::TracerProvider;
 use opentelemetry::trace::TracerProvider as _;
+use opentelemetry_sdk::trace::TracerProvider;
+use opentelemetry_sdk::export::trace::stdout;
```

Comment:
Should revert this, sdk does not have stdout anymore.

Reply:
Where is it located?

Reply:
it's just a standalone crate like it was before`.with_simple_exporter(opentelemetry_stdout::SpanExporter::default())`

Reply:
Yup just realized